qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-pricing-doors-b.scss

```diff
@@ -0,0 +1,33 @@
+@use '../helpers/functions' as *;
+
+.qdrant-pricing-doors-b {
+  padding-bottom: $spacer * 4;
+  padding-top: $spacer * 4;
+  background-color: $neutral-10;
+
+  &__doors {
+    display: flex;
+    flex-direction: column;
+    gap: rem(30);
+    width: 100%;
+  }
+
+  @include media-breakpoint-up(xl) {
+    padding-bottom: $spacer * 5;
+    padding-top: $spacer * 5;
+
+    &__doors {
+      flex-direction: row;
+      justify-content: space-between;
+    }
+
+    &__door {
+      width: calc((100% / 3) - 20px);
```

Comment:
Seems, that A and B variants can be done with the same code if you change `.qdrant-pricing-doors-b__doors` to `justify-content: center;` (see a comment above) and change a little bit this place:

```suggestion
      width: calc((100% / 2) - 20px);
      min-width: calc((100% / 3) - 20px);
      max-width: calc((100% / 2) - 20px);
```

or maybe just using rem(350) and rem(400) for values, depending on what kind of result we want to achieve.

**But no pressure, it's up to you to change it or leave it like it is.**

Reply:
I didn't add this because the height of this text block is different. I can’t think of an option to combine into one component with different block heights

<img width="356" alt="Снимок экрана 2024-04-10 в 16 08 44" src="https://github.com/qdrant/landing_page/assets/38438810/7921474e-9481-4032-bbaa-87179f740b77">
<img width="308" alt="Снимок экрана 2024-04-10 в 16 09 00" src="https://github.com/qdrant/landing_page/assets/38438810/2b2d6ef0-df2c-45c2-9ec2-5a477f3e927d">